yarn-project.nix

```diff
@@ -142,14 +142,13 @@ cacheEntries = {
 "webextension-polyfill@npm:0.8.0" = { filename = "webextension-polyfill-npm-0.8.0-eab1744c30-56286e6216.zip"; sha512 = "56286e6216e15834fdcdc3dcf611000d96b49c9d925de3d657eb5f1e294074180dd3efe78a0966223eb311ac3a0ac5fdf71d324b836bf05724455ffd6c7c98c3"; };
 "buffer@npm:5.7.1" = { filename = "buffer-npm-5.7.1-513ef8259e-e2cf8429e1.zip"; sha512 = "e2cf8429e1c4c7b8cbd30834ac09bd61da46ce35f5c22a78e6c2f04497d6d25541b16881e30a019c6fd3154150650ccee27a308eff3e26229d788bbdeb08ab84"; };
 "@types/jest@npm:26.0.24" = { filename = "@types-jest-npm-26.0.24-47c570294f-ae39675412.zip"; sha512 = "ae39675412f08d884926254e9b12bfd2b5a4e4d204c94d3148cb942174a474930d0c60540133c968f22241d4712b7940c96cbc883096eb326a4d5b206fb78bd0"; };
+"@emurgo/cardano-serialization-lib-browser@npm:11.0.0-rc.6" = { filename = "@emurgo-cardano-serialization-lib-browser-npm-11.0.0-rc.6-f8fcee633c-a5f154735a.zip"; sha512 = "a5f154735a0a4bc9a608bd7dcd431050819dda200ad8824bffb2a49342a83abb0048b5b453ea7090e603df0f883692015eec274921702e4c10e2714f06db363c"; };
```

Comment:
Flagging for review of the nix lock file issue. What determines the order of this file?

![image](https://user-images.githubusercontent.com/303881/185540548-3a97a040-f2f3-4fae-9b61-05876c1cf21c.png)